src/main/java/pages/cartPage/BaseTable.java

```diff
@@ -0,0 +1,58 @@
+package pages.cartPage;
+
+import lombok.Getter;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import pages.BasePage;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+public class BaseTable extends BasePage {
```

Comment:
Do we need to inheritance here?

Reply:
Right. I Changed inheritance to composition
```java
public class BaseTable{
    protected BasePage basePage;
    public String tableXpath;
    public By tableLocator;
    public By headerRow;

    public BaseTable(String tableXpath) {
        this.basePage = new BasePage();
        this.tableXpath = tableXpath;
        this.headerRow = By.xpath(tableXpath + "/tbody/tr");
        this.tableLocator = By.xpath(tableXpath);
    }
}
```